railties/lib/rails/application/configuration.rb

```diff
@@ -191,6 +192,15 @@ def annotations
         SourceAnnotationExtractor::Annotation
       end
 
+      def debug_exception_editor_url=(url)
+        @debug_exception_editor_url = case url
+        when :emacs, :emacsclient then 'emacs://open?url=file://%{file}&line=%{line}'.freeze
+        when :sublime, :subl, :st then 'subl://open?url=file://%{file}&line=%{line}'.freeze
+        when :textmate, :txmt, :tm then 'txmt://open?url=file://%{file}&line=%{line}'.freeze
```

Comment:
Way too many configs. Let's just support the scheme versions by interpreting symbol args as such.